### PR TITLE
roxygen additions and edits; some normalization of function names; minor corrections to isEmptyVar() and fetchWaterQualityMeasurements()

### PR DESCRIPTION
**Pull request by ashalper-usgs**

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -14,6 +14,7 @@ export(getFieldVisitMeasurementsQPoints)
 export(getFieldVisitMeasurementsShifts)
 export(getStatDerived)
 export(getTimeSeries)
+export(isEmptyVar)
```
Comment (mhines-usgs) [inline NAMESPACE]: we probably don't need to export this function, and there are other functions we may not need to export here too. not sure what you changed versus what might be from other people's code.

File: inst/extdata/installDependencies.R [hunk 2]
```diff
@@ -79,7 +79,16 @@ InstallPackages(pkgs, "http://mran.microsoft.com/snapshot/2016-03-31")
 
 # packages that must be held back at older versions
 InstallPackages("httr", "http://mran.microsoft.com/snapshot/2016-01-27")
-InstallPackages("devtools", "http://mran.microsoft.com/snapshot/2015-03-13")
+
```
Comment (mhines-usgs) [inline inst/extdata/installDependencies.R:82]: i'm not sure this is something we want in the code repository? @thongsav-usgs (all the cruft about local things todo)

**Review by mhines-usgs — COMMENTED** [review]
in addition to these small things i commented, it seems like we're inconsistently using @return in the documentation and in functions. not sure if there is a direction we should try to converge upon there.

File: inst/extdata/installDependencies.R (hunk 2 above)
Comment (ashalper-usgs) [reply inst/extdata/installDependencies.R:82]: This is an AQCU-949/AQCU-936 patch that accidentally crept into my `re-awesome` branch. It's obsolete now, and irrelevant anyway, as `installDependencies.R` is not being invoked by Jenkins now. Stay tuned for new-and-improved `installDependencies.R`, as well as invocation by the Jenkins deployer.

File: NAMESPACE (hunk 1 above)
Comment (ashalper-usgs) [reply NAMESPACE]: OK, I removed the @export tag.

File: inst/extdata/installDependencies.R (hunk 2 above)
Comment (mhines-usgs) [reply inst/extdata/installDependencies.R:82]: ahh, good point, i didn't realize this was your .R file, not the .sh file! derp.

Comment (ashalper-usgs) [reply inst/extdata/installDependencies.R:82]: The R script was initially a line-by-line translation of the shell script, so honest mistake! :)

**Comment by thongsav-usgs** [comment]
@ashalper-usgs I'm going to close this as the majority of these files have changed drastically with our big cleanup sprint.

**Pull request closed without merging**